data_set/clean_stackexchanges.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+python clean_stackexchange.py data-collected/stackoverflow-top500.csv https://stackoverflow.com/questions/
+python clean_stackexchange.py data-collected/unixlinux-top500.csv https://unix.stackexchange.com/questions/
+python clean_stackexchange.py data-collected/superuser-top500.csv https://superuser.com/questions/
```

Comment:
Do we want to make this extendable? As in, if we wanted to add sites in the future? 

If so, I would suggest using Bash arrays (or just strings as that's more portable across Bash versions), something along the lines of
```
DATA_DIR="data_collected"
SITES="stackoverflow|unix.stackexchange|superuser"
SITE_TRAILER="questions"
IFS='|' for site in "$SITES"; do
    python clean_stackexchange.py "$DATA_DIR"/"${site}-top500.csv" 'https://'"${site}.com/$SITE_TRAILER"
done
```
This way, adding or changing a site should be relatively easy. 

If not, then the current script should be fine.

Reply:
I love it! I didn't know it was possible to do that in Bash (it would have taken me a lot of time as I'm not fluent in Bash).
I'm integrating it right away. Thank you for the suggestion and the code!